src/stores/AppStore.ts

```diff
@@ -220,6 +220,7 @@ export default class AppStore extends TypedStore {
       if (!url) return;
 
       url = url.replace(/\/$/, '');
+      url = url.replace(/\s?--(updated)/, '');
```

Comment:
Do we want to ensure this "updated" is at the end of the string?

Reply:
@cino what do you think about this?
I think that it should be match regardless of the position... but I don't know the direct implications of that.

Reply:
> so in this case that would work, if electron changes that might not work anymore.

exactly!

Reply:
lets resolve this for now - and cross the bridge if we come to it later.